sdk/highlight-react/src/components/ReportDialog/styles.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.module.css' {
+	const classes: { [key: string]: string }
+	export default classes
+}
```

Comment:
Nit, but perhaps we could use https://www.npmjs.com/package/typed-scss-modules to generate these types in the future if we stick with using CSS modules. I think that's what we use in the main app to keep types in sync for our styles.